src/Http/RequestContext.php

```diff
@@ -118,6 +118,21 @@ public function __construct(PluginClientBuilder $builder, StreamFactoryInterface
         $this->request = $request->withBody($stream);
     }
 
+    /** @When I create the following graphql request: */
+    final public function set_the_graphql_body(string $body): void
+    {
+        $this->query = [];
+        $this->request = $this->requestFactory->createRequest('POST', '/graphql');
```

Comment:
I don't think we should assume that the endpoint is always `/graphql`. Maybe this should be configurable.